client/post-editor/style.scss

```diff
@@ -105,6 +105,10 @@
 	}
 }
 
+.post-editor__drafts-button.drafts-button {
```

Comment:
>Why not do `.post-editor .drafts-button` and avoid adding a className prop?

Partly for consistency, partly for accounting for a potential future case where it exists more than once in the editor. But yes, it would simplify the changes :smile: